github_actions/spec/dependabot/pull_request_creator/labeler_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Dependabot::PullRequestCreator::Labeler do
```

Comment:
There is a test for this class in `common/spec/dependabot/pull_request_creator/labeler_spec.rb`. Maybe these tests should go there?

Reply:
I have moved the test code in https://github.com/dependabot/dependabot-core/pull/4661/commits/a9de16bf24e1550c889ffe61a5a1070cfc1a8e3f.